fair_mango/dataset/dataset.py

```diff
@@ -234,12 +508,81 @@ def get_real_target_for_one_group(self, sensitive: Sequence[str]) -> pd.DataFram
         return result.squeeze()
 
     def get_predicted_target_for_all_groups(self) -> list[dict]:
-        """Retrieve predicted target for all unique groups.
+        """Retrieve the predicted target corresponding to each demographic
+        group present in the sensitive features.
 
```

Comment:
what do you think about wording an example here ?

Suggestion :

```
If you have two sensitive attributes `gender` (that can take the value female, male, non-binary) and `sex` (female, male, other), this function would compute the results for all of the following groups: (female, female), (female, male), (female, other), (male, female), (male, male), (male, other), (non-binary, female), (non-binary, male), (non-binary, other).
```

Reply:
I know, but they are code example, not word example

Reply:
Ah, I got you. I added some word explanations to make it clearer